dotnet/src/Connectors/Connectors.OpenAI/AzureSdk/OpenAIFunctionToolCall.cs

```diff
@@ -176,5 +176,5 @@ internal static ChatCompletionsFunctionToolCall CreateChatCompletionsFunctionToo
         s_type?.SetValue(c, "function");
         return c;
     }
-    private static readonly PropertyInfo? s_type = typeof(ChatCompletionsToolCall).GetProperty("Type", BindingFlags.Instance | BindingFlags.NonPublic | BindingFlags.Public);
+    private static readonly PropertyInfo? s_type = typeof(ChatCompletionsFunctionToolCall).GetProperty("Type", BindingFlags.Instance | BindingFlags.NonPublic | BindingFlags.Public);
```

Comment:
Was this change necessary? I wouldn't expect it to be.

Reply:
Reverted